spec/iss36-spec.coffee

```diff
@@ -34,10 +34,10 @@ describe 'linter-javac', ->
     it 'should detect all errors and warnings', ->
       expect(@linterBase.parse(@linter_output, @textEditor).length).toBe(4)
 
-    it 'should translate the first message-type to error', ->
+    it 'should translate the first message-severity to error', ->
       firstMessage = @linterBase.parse(@linter_output, @textEditor)[0]
-      expect(firstMessage.type).toMatch('error')
+      expect(firstMessage.severity).toMatch('error')
```

Comment:
They **must** be exact matches, so this should be `toBe` instead of `toMatch`.

Reply:
ouch..